.github/workflows/release.yml

```diff
@@ -7,7 +7,9 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    # Use older Ubuntu release to avoid building with a too new glibc, which will cause
+    # errors on older distributions, if anyone uses the generated binary directly.
```

Comment:
Is the converse not a problem? That is, using the older-build binary on newer distributions?

Reply:
Works fine - glibc is a minimum requirement (based on the version its built with), but you can have a newer glibc release on the host and it will know what to do.